### PR TITLE
Belarusian: improvements

### PR DESCRIPTION
**Pull request by bespsm**
- fixed/removed not working rules in be_list
- added stress to the words in be_list
- fixed multi thousand transcription
- removed not working rules in be_rules
- added rules of palatalization, phonemes lengthen
- fixed dropping of [a] at the end of words
- fixed message "Full dictionary is not installed for"
- added configuration in tr_languages.c
- fixed/added phonemes for `Q`, `ts`, `ts;`, `dz`, `dz.`, `;` etc

added tests to:
- language-numbers-cardinal.test
- tests/language-phonemes.test
- tests/language-pronunciation.test

based on: 
https://baltoslav.eu/ipa/index.php?mova=en&j=by
https://corpus.by/TranscriptionGenerator/?lang=be

### CODE REVIEW

File: dictsource/be_rules [hunk 1]
```diff
@@ -36,35 +36,35 @@
...
 
 .group ё
-       ё       ;'O    // ʲɔ
-    _) ё       j'O
-    ') ё       j'O
-    ь) ё       j'O
-  L01) ё       j'O
+       ё       ;O    // ʲɔ
+ // _) ё       jO    // applies not only to start of the word
+    ') ё       ;jO
```
Comment (HaraldZealot) [inline dictsource/be_rules:54]: you swapped the behavior for apostrophe and soft sign

Comment (bespsm) [reply dictsource/be_rules]: fixed

File: ChangeLog.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ The espeak-ng project is a fork of the espeak project.
 
 updated languages:
 *  ba (Bashkir) -- Andiv06
-*  be (Belarusian) -- Andiv06
+*  be (Belarusian) -- Andiv06, bespsm, HaraldZealot
```
Comment (HaraldZealot) [inline ChangeLog.md:14]: Don't you want use real names? I'm Alaksiej Stankievič

File: tests/language-pronunciation.test [hunk 3]
```diff
@@ -19,6 +19,11 @@ X'ihO~t,Or b'ibek 'a*u b'udd#i 'atS#e 'a*u X'ihO~t,e p'OrOXp,Or b#atr'itbO*,e 'a
 test_phon az "byts'yn insanL'aR l&jag'&t v& hygugLa*@n'a J'W*& az'ad b&*ab'&R do:uLuRL'aR
 ona*'@n Syu*aLR'@ v& vidZdanLa*'@ v'aR v& b'iRbiRl&*'in& mynasib'&td& gaRdaSL'@x Runhund'a davRanmaL@d@RL'aR" "Bütün insanlar ləyaqət və hüquqlarına görə azad bərabər doğulurlar. Onarın şüuralrı və vicdanları var və bir-birlərinə münasibətdə qardaşlıq runhunda davranmalıdırlar." "Latn"
 test_phon ba "m'in h'in 'uL b'eD h'eD 'uLAr b'VL t'eQe" "мин һин ул беҙ һеҙ улар был теге." "Cyrl"
+test_phon be "Qr'us.a tsv;'ila ap'Os.n;i Q'Od
```
Comment (HaraldZealot) [inline tests/language-pronunciation.test:22]: `tsv;'ila` -> `ts;v;il'a`, `Q'Od` -> `Q'Ot`

File: tests/language-pronunciation.test [hunk 4]
```diff
@@ -19,6 +19,11 @@ X'ihO~t,Or b'ibek 'a*u b'udd#i 'atS#e 'a*u X'ihO~t,e p'OrOXp,Or b#atr'itbO*,e 'a
...
+'us;E Q'al;ini\" j'ajE
```
Comment (HaraldZealot) [inline tests/language-pronunciation.test:23]: Is it automatically generated? as all stresses are wrong

Comment (bespsm) [reply tests/language-pronunciation.test:23]: exactly, so this will be our next task to implement and add these rules, but now it works this way

**Comment by HaraldZealot** [comment]
LGTM

**Comment by bespsm** [comment]
review is finished, please check/merge it, @valdisvi @Andiv06

**Pull request merged**